.github/actions/prettier/action.yml

```diff
@@ -0,0 +1,25 @@
+# Define action name, description, and inputs
+name: "Prettier Lint"
+description: "Lint code with Prettier"
+inputs:
+  working-directory:
+    required: false
+    description: "Working directory"
+    default: "."
+
+# Define action steps
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 16.x
+    - run: |
+        yarn
+        yarn prettier --check .
+      shell: bash
+      working-directory: ${{ inputs.working-directory }}
+    - if: inputs.working-directory == 'packages/dashboard-v2' || inputs.working-directory == 'packages/dashboard'
+      run: yarn lint
```

Comment:
@kwypchlo is this outdated? The action returns the error that no command exists for `yarn lint`. I get the same error locally.

Reply:
```
inovakov dashboard-v2 [master]% yarn lint
yarn run v1.22.18
$ eslint .
✨  Done in 7.99s.
```
I had to go into `skynet-webportal/packages/dashboard-v2` and install npm modules but it worked like a charm.